src/domain/relay/errors/unofficial-mastercopy.error.ts

```diff
@@ -0,0 +1,7 @@
+export class UnofficialMastercopyError extends Error {
```

Comment:
nitpick: `UnofficialMasterCopyError`

Reply:
Changed in 4a9d3787f8f6f122c9eb31a6e5e4cb00acc1c6e4.